io.github.chilledheart.yass.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<component type="desktop">
+  <id>io.github.chilledheart.yass</id>
+  <name>YASS</name>
+  <developer id="letshack.info">
+    <name>Chilledheart</name>
+  </developer>
+  <developer_name>Chilledheart</developer_name>
+  <summary>Lightweight and efficient, socks/http forward proxy</summary>
+  <description>
+    <p>Free and open source software for socks/http forward proxy including http2/naiveproxy/https/socks4/socks5/shadowsock protocols</p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <url type="homepage">https://github.com/Chilledheart/yass</url>
+  <launchable type="desktop-id">io.github.chilledheart.yass.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://github.com/Chilledheart/yass/assets/54673341/4e2a2e47-43fb-402f-90c7-d2e033cc6a1d</image>
+      <caption>Main window</caption>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="1.12.2" date="2024-08-07" type="stable"/>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
```

Comment:
```suggestion
  <content_rating type="oars-1.1" />
```

Reply:
Done in 0e795974845a11ab2cab91c9762b9995a66c7f76.